src/editor.cpp

```diff
@@ -66,7 +72,6 @@ void EditorWidget::contextMenuEvent(QContextMenuEvent *e)
 
 void EditorWidget::finalizeInitialization()
 {
-    // TODO finalizeInitializationAfterDuplication
```

Comment:
what about rest of things that happen in this function? shouldn't they also be applied "after duplication" ?

Reply:
It seems that function `finalizeInitialization` is called in both cases: when the object is created and copied:
```
finalizeInitialization this:   Asn1Acn::Internal::AsnEditorWidget(0x55c299507640)
finalizeInitialization this:   Asn1Acn::Internal::AsnEditorWidget(0x55c2993a8960)
finalizeInitializationAfterDuplication this:  Asn1Acn::Internal::AsnEditorWidget(0x55c2993a8960) other:  Asn1Acn::Internal::AsnEditorWidget(0x55c299507640)
```
`finalizeInitializationAfterDuplication` turns out to be for action needed by copied object exclusively. The question however, leaded me to notice other mistake in code, so please note the another commit pushed.